docs/modules/ROOT/pages/contributor-guide/ci.adoc

```diff
@@ -11,6 +11,28 @@
 |quarkus-master| To verify camel-quarkus against the latest Quarkus SNAPSHOT release
 |===
 
+=== Branch maintenance
+
+Some scheduled build jobs (mentioned below) run each day to synchronize branches `camel-master` & `quarkus-master` with the latest work from the `master` branch.
+
+Sometimes these jobs fail and it is necessary to fix any issues and manually rebase the branches. The process for doing this is as follows.
+
+1. Depending on which branch you're rebasing, you may want to begin by building the latest Camel `master` or Quarkus `main` branch to avoid pulling in non-deterministic SNAPSHOT artifacts. Otherwise you can build the branches and activate the relevant SNAPSHOT repositories via Maven profiles `-Papache-snapshots` or `-Poss-snapshots`.
```

Comment:
Pro tip: Build Quarkus with `mvnd clean install -Dquickly` and Camel with `mvnd clean install -Pfastinstall`

Reply:
maybe one day we should use the same profile anme across the projects :)